### DISCUSSION TITLE
localStorage is not defined  [ cli@17

### DISCUSSION THREAD

**Discussion by flcrepaldi** [po-ui/po-angular#2038] [Angular] [answered]
Bom dia.

Estou subindo uma aplicação do Zero.
Node.Js : v20.12.2  

![image](https://github.com/po-ui/po-angular/assets/74430333/15db32fb-37d0-49c9-823c-658fc406fa2b)

Instalei com o NPM o Angular 17 (  npm i -g @angular/cli@17 ) 
 
Criei o projeto  ( ng new my-po-project --skip-install)
Ajustei  a  lista de dependências e as versões compatíveis 
![image](https://github.com/po-ui/po-angular/assets/74430333/ad50b987-6714-4b15-a60f-ceed2b454bfd)

Logo apos, fiz a instalação:  npm install e adicionei o pacote @po-ui/ng-components, tudo sem erros.

Subo a aplicação via ng serve sem erros

![image](https://github.com/po-ui/po-angular/assets/74430333/2a3a5e5c-e6a1-4b0f-af94-85a247fb8669)
![image](https://github.com/po-ui/po-angular/assets/74430333/a5349e7b-f9b7-48f6-95cb-25ca064195db)

**Porem, quando vou acessar é apresentado erro abaixo:**

![image](https://github.com/po-ui/po-angular/assets/74430333/d5251d13-6b39-4b20-94b6-732b8296f93e)
![image](https://github.com/po-ui/po-angular/assets/74430333/e7a44ccf-7653-410e-ab47-1ee4b7b4a1ba)

Alguem consegue me ajudar a identificar o problema?

**Comment by anliben** [comment]
Boa Tarde.

Poderia verificar seu ambiente e nos enviar uma evidência?
Não conseguimos identificar a inconsistência.

**Comment by flcrepaldi** [comment]
Boa tarde.

Verificar exatamente oq ?
Eu removi tudo e comecei varias vezes e sempre tenho esse mesmo problema.
E só acontece se eu adiciono @po-ui/ng-components .  Sem o @po-ui/ng-components ele sobe normal a aplicação.

**Comment by PeterNewtonBR** [comment, marked as answer]
Bom dia,

Comigo estava ocorrendo o mesmo, fiaz várias instalações e sempre o mesmo problema.
Daí fiz a instalação sem o uso do SSR. Quando vai instalar, o programa pergunta se deseja a instalação do "Server-side rendering (SSR)".
Fiz sem ele e deu certo. Tudo versão nova ocorreu esse conflito.

**Reply by flcrepaldi** [reply]
Bom dia.

Vou fazer, muito obrigado pela dica.
